addons/l10n_ch/models/res_bank.py

```diff
@@ -164,43 +92,6 @@ def write(self, vals):
             vals['l10n_ch_qr_iban'] = pretty_iban(normalize_iban(vals['l10n_ch_qr_iban']))
         return super().write(vals)
 
-    @api.model
-    def _get_supported_account_types(self):
-        rslt = super(ResPartnerBank, self)._get_supported_account_types()
-        rslt.append(('postal', _('Postal')))
-        return rslt
-
-    @api.model
-    def retrieve_acc_type(self, acc_number):
-        """ Overridden method enabling the recognition of swiss postal bank
-        account numbers.
-        """
-        acc_number_split = ""
-        # acc_number_split is needed to continue to recognize the account
-        # as a postal account even if the difference
-        if acc_number and " " in acc_number:
-            acc_number_split = acc_number.split(" ")[0]
-        if _is_l10n_ch_postal(acc_number) or (acc_number_split and _is_l10n_ch_postal(acc_number_split)):
-            return 'postal'
-        else:
-            return super(ResPartnerBank, self).retrieve_acc_type(acc_number)
-
-    @api.depends('acc_number', 'partner_id', 'acc_type')
-    def _compute_l10n_ch_postal(self):
-        for record in self:
-            if record.acc_type == 'iban':
-                record.l10n_ch_postal = self._retrieve_l10n_ch_postal(record.sanitized_acc_number)
-            elif record.acc_type == 'postal':
-                if record.acc_number and " " in record.acc_number:
-                    record.l10n_ch_postal = record.acc_number.split(" ")[0]
-                else:
-                    record.l10n_ch_postal = record.acc_number
-                    # In case of ISR issuer, this number is not
-                    # unique and we fill acc_number with partner
-                    # name to give proper information to the user
-                    if record.partner_id and record.acc_number[:2] in ["01", "03"]:
-                        record.acc_number = ("{} {}").format(record.acc_number, record.partner_id.name)
-
     @api.model
     def _is_postfinance_iban(self, iban):
```

Comment:
This can be dropped as well. It was just an helper to know when we could compute the postal number from the IBAN in `_retrieve_l10n_ch_postal`.